plugins/GSdx/Renderers/HW/GSRendererHW.cpp

```diff
@@ -1689,6 +1690,38 @@ bool GSRendererHW::OI_BlitFMV(GSTextureCache::Target* _rt, GSTextureCache::Sourc
 
 // OI (others input?/implementation?) hacks replace current draw call
 
+bool GSRendererHW::OI_BigMuthaTruckers(GSTexture* rt, GSTexture* ds, GSTextureCache::Source* t)
+{
+	// Rendering pattern is:
+	// Front buffer(CRTC) 0x0 is sent to TV with interlaced resolution w, h/2. Only RGB channels are useful.
+	// The game needs to do a depth effect so it needs to move Green channel to an Alpha channel.
+	// To get around GS memory limitations Green is moved to the Alpha of front buffer,
+	// but resolution is smaller so only half of the screen gets processed at once.
+
+	// Solution as we don't have this memory limitation just do full screen instead of half, 
+	// however to completely solve any half screen issues would be to properly detect bottom half at TC level.
+
+	GIFRegTEX0 Texture = m_context->TEX0;
+
+	GIFRegTEX0 Frame;
+	Frame.TBW = m_context->FRAME.FBW;
+	Frame.TBP0 = m_context->FRAME.FBP;
+	Frame.TBP0 = m_context->FRAME.Block();
+
+	if (PRIM->TME && Frame.TBW == 10 && Texture.TBW == 10 && Frame.TBP0 == 0x00a00 && Texture.PSM == PSM_PSMT8H && (m_r.y == 256 || m_r.y == 224))
+	{
+		// 224 ntsc, 256 pal.
+		GL_INS("OI_BigMuthaTruckers half bottom offset");
+		size_t count = m_vertex.next;
+		GSVertex* v = &m_vertex.buff[0];
+		uint16 offset = (uint16)m_r.y * 16;
+		for (size_t i = 0; i < count; i++)
```

Comment:
Add some space maybe
```cpp
	    GL_INS("OI_BigMuthaTruckers half bottom offset");

		size_t count = m_vertex.next;
		GSVertex* v = &m_vertex.buff[0];
		uint16 offset = (uint16)m_r.y * 16;

		for (size_t i = 0; i < count; i++)
```

Just a little easier for me to read. Otherwise fine by me.

Reply:
Actually now that I look at it you can probably throw some `const` in there.
Not required but if you want to go ahead.